hydrolib/core/basemodel.py

```diff
@@ -107,6 +107,38 @@ def _get_identifier(self, data: dict) -> Optional[str]:
         """
         return None
 
+    @classmethod
+    def get_list_delimiter(cls) -> str:
+        """List delimiter string that will be used for serializing
+        list field values for any pydantic ModelField in this BaseModel,
+        **if** that field has no custom list delimiter.
+
+        This function should be overridden by any subclass for a particular
+        filetype that needs a specific/different list separator.
+        """
+        # Dummy default for this BaseModel class
+        return ","
+
+    @classmethod
+    def get_list_field_delimiter(cls, field_key: str) -> str:
```

Comment:
As discussed I have a question, whether these functions should be on the `BaseModel` or the `INIBasedModel` level?

Reply:
Done, moved to INIBasedModel.